### PR TITLE
fix: fallback to stock uom if uom is not defined

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
**Issue**

If UOM is not added then getting below error
```
Traceback (most recent call last):
  File "/home/frappe/benches/bench-2018-11-15/apps/frappe/frappe/app.py", line 62, in application
    response = frappe.handler.handle()
  File "/home/frappe/benches/bench-2018-11-15/apps/frappe/frappe/handler.py", line 22, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/benches/bench-2018-11-15/apps/frappe/frappe/handler.py", line 53, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/benches/bench-2018-11-15/apps/frappe/frappe/__init__.py", line 939, in call
    return fn(*args, **newargs)
  File "/home/frappe/benches/bench-2018-11-15/apps/erpnext/erpnext/stock/get_item_details.py", line 44, in get_item_details
    out = get_basic_details(args, item)
  File "/home/frappe/benches/bench-2018-11-15/apps/erpnext/erpnext/stock/get_item_details.py", line 261, in get_basic_details
    out.stock_qty = out.qty * out.conversion_factor
TypeError: can't multiply sequence by non-int of type 'float'
```

### CODE REVIEW

**Comment by sagarvora** [comment]
Please review PR title. It's a bug fix and same must be reflected.

File: erpnext/stock/get_item_details.py [hunk 1]
```diff
@@ -281,7 +281,7 @@ def get_basic_details(args, item):
 		out.conversion_factor = 1.0
 	else:
 		out.conversion_factor = args.conversion_factor or \
-			get_conversion_factor(item.item_code, args.uom).get("conversion_factor") or 1.0
+			get_conversion_factor(item.item_code, args.uom or item.get('stock_uom')).get("conversion_factor", 1) or 1.0
```
Comment (sagarvora) [inline erpnext/stock/get_item_details.py]:
```suggestion
			get_conversion_factor(item.item_code, args.uom or item.get('stock_uom')).get("conversion_factor") or 1.0
```

seeing as how we already have `or 1.0`

**Comment by sagarvora** [comment]
Why does the error try to multiply a sequence. Pls check if `out.qty` is a sequence, not float.

**Comment by sagarvora** [comment]
@rohitwaghchaure I think that it was a non-empty string. Considering that if it were an empty string, it should execute the code after `or` and default to `1.0`
Let's use `flt` to solve this.

**Review by saurabh6790 — APPROVED** [review]

**Pull request merged**